components/prism-renpy.js

```diff
@@ -0,0 +1,28 @@
+Prism.languages.renpy= {
+
+	'comment': {
+		pattern: /(^|[^\\])#.*?(\r?\n|$)/,
+		lookbehind: true
+	},
+
+	'string': /"""[\s\S]+?"""|'''[\s\S]+?'''|("|')(\\?.)*?\1|(^#?(?:(?:[0-9a-fA-F]{2}){3}|(?:[0-9a-fA-F]){3})$)/,
+
+	'function' : {
+		pattern: /([a-zA-Z_][a-zA-Z0-9_]*(?=\())/g,
+		lookbehind: false
```

Comment:
Lookbehind is `false` by default, so you can simplify the `function` value by just providing the regexp.